.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: objective-c
-osx_image: xcode7.3
+osx_image: xcode8.0
```

Comment:
Sure. I think we kept the 9.3 SDK to maintain the check in iOS 9 so we know it's still compatible with iOS 9